Sources/Controllers/Map/Widgets/OAMapWidgetRegistry.mm

```diff
@@ -351,9 +351,8 @@ - (OAMapWidgetInfo *)getWidgetInfoForType:(OAWidgetType *)widgetType
             BOOL passAvailable = !availableMode || [OAWidgetsAvailabilityHelper isWidgetAvailableWithWidgetId:widget.key appMode:appMode];
             BOOL defaultAvailable = !defaultMode || !widget.isCustomWidget;
             BOOL passMatchedPanels = !matchingPanelsMode || [panels containsObject:widget.widgetPanel];
-            BOOL passTypeAllowed = [widget getWidgetType] == nil || [[widget getWidgetType] isAllowed];
```

Comment:
Is this feature already implemented on Android? We are synchronizing the flow of widgets with Android. I see they have such code as well